docs/platform/concepts/service_backups.md

```diff
@@ -62,6 +62,8 @@ backups with the appropriate tooling:
     `cqlsh`
 -   [OpenSearch®](https://github.com/elasticsearch-dump/elasticsearch-dump):
     `elasticdump`
+-   [Dragonfly®](https://www.dragonflydb.io/docs/development/cli): `dragonfly-cli`
```

Comment:
there's no `dragonfly-cli`, either `redis-cli` or `valkey-cli` would work. I see they recommend `redis-cli` in the docs, so we can go with that